platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/StateSignatureCollectorWiring.java

```diff
@@ -48,33 +59,55 @@ public class StateSignatureCollectorWiring {
      * @param taskScheduler the task scheduler that will perform the prehandling
      */
     private StateSignatureCollectorWiring(
-            @NonNull final WiringModel model, @NonNull final TaskScheduler<Void> taskScheduler) {
+            @NonNull final WiringModel model, @NonNull final TaskScheduler<List<ReservedSignedState>> taskScheduler) {
 
         this.taskScheduler = Objects.requireNonNull(taskScheduler);
-
-        final WireTransformer<GossipEvent, List<ScopedSystemTransaction<?>>> systemTransactionsFilter =
-                new WireTransformer<>(
+        // Create the outputs
+        final OutputWire<ReservedSignedState> stateSplitter =
+                taskScheduler.getOutputWire().buildSplitter("reservedStateSplitter", "reserved states");
+        this.allStatesOutput = stateSplitter.buildAdvancedTransformer(new SignedStateReserver("allStatesReserver"));
+        this.completeStatesOutput = allStatesOutput
+                .buildFilter(
+                        "filterOnlyCompleteStates",
+                        "reservedStateOutput",
+                        StateSignatureCollectorWiring::completeStates)
+                .buildAdvancedTransformer(new SignedStateReserver("completeStatesReserver"));
+
+        // Create input for pre-consensus signatures
+        final WireTransformer<GossipEvent, List<ScopedSystemTransaction<StateSignatureTransaction>>>
+                preConsensusTransformer = new WireTransformer<>(
                         model,
-                        "systemTransactionsFilter",
-                        "preconsensus events",
-                        SystemTransactionExtractor::getScopedSystemTransactions);
-        preconsensusEventInput = systemTransactionsFilter.getInputWire();
-
-        final WireListSplitter<ScopedSystemTransaction<?>> systemTransactionsSplitter =
-                new WireListSplitter<>(model, "systemTransactionsSplitter", "system transaction lists");
-
-        final WireTransformer<ScopedSystemTransaction<?>, ScopedSystemTransaction<StateSignatureTransaction>>
-                stateSignatureTransactionFilter = new WireTransformer<>(
+                        "preConsensusTransformer",
+                        "pre-consensus events",
```

Comment:
Should this string be updated with @cody-littley suggestion?

Reply:
yes, missed that one